Lib/multiprocessing/connection.py

```diff
@@ -329,10 +337,15 @@ def _recv_bytes(self, maxsize=None):
             raise RuntimeError("shouldn't get here; expected KeyboardInterrupt")
 
         def _poll(self, timeout):
-            if (self._got_empty_message or
-                        _winapi.PeekNamedPipe(self._handle)[0] != 0):
-                return True
-            return bool(wait([self], timeout))
+            try:
+                if (self._got_empty_message or
+                            _winapi.PeekNamedPipe(self._handle)[0] != 0):
+                    return True
+                return bool(wait([self], timeout))
```

Comment:
Does it work if you pass `self._handle` to `wait()`?

Reply:
Yes it works